crates/simplexpr/src/eval.rs

```diff
@@ -91,6 +91,9 @@ impl SimplExpr {
                     .collect::<Result<_, _>>()?,
             ),
             x @ Literal(..) => x,
+            LetIn(_, d, b) => {
+                todo!("Resolve Refs is unused, TODO(josiah)");
```

Comment:
It's unclear to me atm what should be done here. As far as I can tell the function is (transitively) unused, and could be removed, so it might not be a problem.

It might make the most sense to use the values from the let block to map the inner vars, but the variable defs haven't been evaluated yet here.